release-notes/6.0/preview/api-diff/preview6/.Net/6.0-preview6_System.Runtime.Intrinsics.X86.md

```diff
@@ -0,0 +1,21 @@
+# System.Runtime.Intrinsics.X86
+
+``` diff
```

Comment:
Intrinsics LGTM. CC @echesakovMSFT, @pgovind 

Do we need to do anything explicit to express these are `RequiresPreviewFeatures`?
The attribute is in the ref but isn't relayed here and likely not readily visible in the docs/etc

CC. @terrajobst, @carlossanlop, @jeffhandley  ^

Reply:
I created https://github.com/dotnet/dotnet-api-docs/issues/6861 for this.

Reply:
@Anipik, can we ensure that a particular attribute is included in the future diffs (namely `RequiresPreviewFeature`)?